content/_help/manage-your-account/personal-key._en.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: help
+title: Personal key
+category: manage-your-account
+permalink: /help/manage-your-account/personal-key/
+order: 3 
+redirect_from:
+  - /en/help/manage-your-account/personal-key/
+---
+A personal key is a 16-character code that encrypts your personal information with your account. Your personal key is the only way to access your information if you lose or forget your password.
+
+Keep in mind:
+
+- You’ll need to verify your identity again if you reset your password without your personal key. 
+- Personal keys can only be used one time. If you’ve used a personal key to reset your password before, you have a new personal key. 
+- You can find your current personal key on your account page.
+
+You get your personal key when you verify your identity. Save and keep your personal key in a safe place. You can copy, download, or print it.
+
+## Changing or resetting your password
+
+### If you have a personal key 
+
+When you reset your password, you’ll need your personal key to access your information. 
+
+1. Once you have reset your password, sign in using your new password.
+2. You will see a message on your account page alerting you that your account was recently deactivated due to a password reset.
+3. Select “Reactivate your profile now.” 
+4. Select “I have my key”
+5. Enter your most recent personal key. If you’ve reset your password in the past, you may have a new personal key.
+6. Re-enter your password.
+7. Be sure to save your new personal key after you reset your password.
+
+### If you don’t have a personal key
+
+If you reset your password without your personal key, you’ll need to verify your identity again.
```

Comment:
Is it worth specifying here that if they change their password while logged in, this doesn't apply?